package.json

```diff
@@ -2,7 +2,7 @@
   "name": "web.dev",
   "version": "2.0.0",
   "scripts": {
-    "build": "npm-run-all clean sass gulp rollup eleventy rollup-sw",
+    "build": "npm-run-all clean sass gulp rollup eleventy sw-manifest",
```

Comment:
Doesn't rollup consume the manifest created by sw-manifest? Should the order be `clean sass gulp sw-manifest rollup eleventy`?

Reply:
ah I see, because this script is loaded at runtime instead of being injected then it can come at the very end.